src/cpp/library/string

```diff
@@ -991,7 +987,7 @@ namespace std
 string::string()
 {
   str = new char[1];
-  str = "";
+  str[0] = '\0'; // Assign the null terminator to represent an empty string
```

Comment:
Not sure about this. In C++ [string::empty](https://cplusplus.com/reference/string/string/empty/) means a string object with size 0. I am not even sure why we are allocating something to it.

Reply:
Fixed.